src/Jackett.Common/Definitions/matrix.yml

```diff
@@ -0,0 +1,175 @@
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: 0
+    options:
+      0: created
+      1: title
+      4: size
+      6: seeders
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+  - name: info_tpp
+    type: info
+    label: Results Per Page
+    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.
+
+login:
+  path: takelogin.php
+  method: post
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    logout: ""
+  error:
+    - selector: td.embedded:contains("Sikertelen bejelentkezés")
+  test:
+    path: /
+    selector: a[href$="logout.php"]
+
+search:
+  paths:
+    - path: browse.php
+  inputs:
+    $raw: "{{ range .Categories }}c{{.}}=1&{{end}}"
+    search: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
+    # 0 active only, 1 include dead, 2 only dead
+    incldead: 1
+    sort: "{{ .Config.sort }}"
+    type: "{{ .Config.type }}"
+
+  rows:
+    selector: "table.mainouter > tbody > tr > td > table tr:has(a[href^=\"download.php?id=\"]){{ if .Config.freeleech }}:has(img[src$=\"/pic/ingyentorrent.gif\"]){{ else }}{{ end }}"
+
+  fields:
+    category:
+      selector: a[href^="browse.php?cat="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: cat
+    title:
+      selector: a.torrent-link b
+    _id:
+      selector: a[href^="download.php?id="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: id
+    details:
+      text: "details.php?id={{ .Result._id }}"
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    poster:
+      selector: a.torrent-link
+      attribute: data-cover
+    imdbid:
+      selector: a[href*="imdb.com/title/tt"]
+      attribute: href
+    files:
+      selector: td:nth-child(5) a
+    date:
+      selector: td:contains("Feltöltve:")
+      filters:
+        - name: regexp
+          args: "Feltöltve: *([0-9]{4}-[0-9]{2}-[0-9]{2} +[0-9]{2}:[0-9]{2}:[0-9]{2})"
+        - name: append
+          args: " +01:00" # Europe/Budapest, CET
+        - name: dateparse
+          args: "yyyy-MM-dd HH:mm:ss zzz"
+    grabs:
+      selector: td:nth-child(7) b
+    size:
+      selector: td:nth-child(7)
+      filters:
+        - name: regexp
+          args: "([0-9]+\\.?[0-9]* *[kKmMgGtTpP]i?[bB])"
+    seeders:
+      selector: td:nth-child(8)
+    leechers:
+      selector: td:nth-child(9)
+    genre:
+      selector: td:nth-child(2) > div > i > i
+      filters:
+        - name: regexp
+          args: "\\((.+)\\)"
+    description:
+      text: "{{ .Result.genre }}"
+    downloadvolumefactor:
+      case:
+        img[src$="/pic/ingyentorrent.gif"]: 0
+        "*": 1
+    uploadvolumefactor:
+      case:
+        img[src$="/pic/x2.gif"]: 2
+        "*": 1
```

Comment:
```suggestion
        "*": 1
    minimumratio:
      text: 1.0
    minimumseedtime:
      # 2 days (as seconds = 2 x 24 x 60 x 60)
      text: 172800
```

Reply:
Hehe, that's an embarrassing miss on my part. I discovered those two fields recently and it's what got me more interested in working with indexer definitions. :-/ Thanks!

Reply:
I can't confirm this fix because I don't see any corresponding fields in the Prowlar search results UI nor in XHR JSON. How can we confirm this change?

Reply:
It's information that is sent to the torrent client when a torrent/magnet is downloaded, and those limits are applied to the torrent (assuming the client supports it). There's no need to check it when just making/updating an indexer, it'd be more if you were making some core change to Jackett or Prowlarr.

The only time you'd need to check anything about it for an indexer is if you were having to use selectors, e.g. freeleech=3d, else=2d, and even then just make sure the selectors are ok.

Reply:
K, thanks for explaining. Looks like we're good on this change too, so I think I've integrated and tested all changes so far.